packages/backend/src/api/controllers/FrontendController.ts

```diff
@@ -117,6 +117,8 @@ export class FrontendController {
       html: renderStateUpdateDetailsPage({
         id: stateUpdate.id,
         hash: stateUpdate.hash,
+        rootHash: PedersenHash.fake(), // TODO: use real value
+        blockNumber: 1000n, // TODO: use real value
```

Comment:
These two will be added once https://github.com/l2beat/dydx-state-explorer/pull/76 is merged since aforementioned PR partially solves this.